backend/question_service/middleware/isAdminCheck.js

```diff
@@ -0,0 +1,37 @@
+const isAdminCheck = async (req, res, next) => {
+    // Extract the token from the request headers
+    const token = req.headers['authorization']?.split(' ')[1];
+    
+    if (!token) {
+        return res.status(401).json({ error: "Authorization token is required." });
```

Comment:
`error` label should be changed to `message` to fit how the frontend handles it.